planet/clients/orders.py

```diff
@@ -302,68 +302,51 @@ async def aggregated_order_stats(self) -> dict:
         resp = await self._do_request(req)
         return resp.json()
 
-    async def download_asset(self,
-                             location: str,
-                             filename: str = None,
-                             directory: str = None,
+    async def download_order(self,
+                             order: Order,
+                             directory: Path,
```

Comment:
I'm experimenting with taking order (to save an extra web request) and Python Path object.

Reply:
To avoid unpleasant surprises, I believe we should always require a directory to save to and not default to the current working directory. I think this is more in line with Python APIs.